src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,40 @@
+package parser;
+
+import task.Task;
+import task.ToDo;
+import task.Deadline;
+import task.Event;
+
+public class Parser {
+
+    public static Task parseTaskFromSaveFormat(String line) {
```

Comment:
Many magic literals here, some comments would be helpful for the reader to understand the logic here.